contracts/core/modules/RebalanceAuctionModule.sol

```diff
@@ -126,11 +139,24 @@ contract RebalanceAuctionModule is
      */
     function bidAndWithdraw(
         address _rebalancingSetToken,
-        uint256 _quantity
+        uint256 _quantity,
+        bool _executePartialQuantity
     )
         external
         nonReentrant
     {
+        // If user allows partial fills calculate partial fill (if necessary)
+        uint256 executionQuantity;
+        if (_executePartialQuantity) {
```

Comment:
We can consolidate this logic into a function like `getExecutionQuantity(_quantity, _executePartialQuantity)` that just returns the execution quantity.